### ISSUE TITLE
The main interface of the website

### ISSUE THREAD

**Issue by mohannadhanafi** [FACG5/Hacker-News#26]
The main interface of the website doesn't have anything.
So as normal user I can't deal with it.

**Comment by donia98** [comment]
yeah i agree with you .. I'll try to make it easy to deal with ..
